packages/core/echo/automerge/package.json

```diff
@@ -149,6 +149,7 @@
     "eventemitter3": "^5.0.1",
     "fast-sha256": "^1.3.0",
     "isomorphic-ws": "^5.0.0",
+    "uuid": "^9.0.0",
```

Comment:
we're not able to upgrade our usage of uuid?

Reply:
No, did not try it

Reply:
I would guess that it would be fine, I aligned them at v8 previously after noticing that we had multiple copies in the bundle and it was fairly simple upgrade iirc so probably worth a shot